getting_started/azure_vm/install_ccf_on_azure_vm.sh

```diff
@@ -0,0 +1,38 @@
+# Change these
+resourceGroup=rg_ccfdemo
+vm_name=ccfdemo
+vm_dns=ccfdemo-vm-sample
+
+# These are defaults
+location=westeurope
+vm_size=Standard_DC1s_v2
```

Comment:
This seems pretty small for the default, see https://learn.microsoft.com/en-us/azure/virtual-machines/dcv3-series#dcsv3-series. Perhaps `Standard_DC16s_v3` would be a better default here?

Reply:
To run, rather than build CCF, this seems fine?

Reply:
I was able to run the banking sample on a DC1s. I've added a comment to tell people that this is the smallest VM. I didn't want users to pick a large machine and leave it running for a getting started

Reply:
@ross-p-smith this is a good choice to get started, they can scale up if/when they need to.